doc/sphinx/reactors.rst

```diff
@@ -1,74 +1,38 @@
 .. default-role:: math
 
-****************
-Reactor Networks
-****************
-
-Cantera's Reactor Network module is designed to simulate networks of
-interconnected reactors. The contents of each reactor in the network are
-assumed to be homogeneous, a model variously referred to as the Continuously
-Stirred Tank Reactor (CSTR), Well-Stirred Reactor (WSR), or Perfectly Stirred
-Reactor (PSR) model. Cantera solves the time-dependent governing equations
-that describe the evolution of the chemical and thermodynamic state of the
-reactors.
-
-The contents of each reactor can undergo chemical reactions according to a
-specified kinetic mechanism, and surface reactions may occur on the reactor
-walls. Each reactor in a network may be connected so that the contents of one
-reactor flow into another. Reactors may be also be in contact with one another
-or the environment via walls which move or conduct heat.
-
-The purpose of this document is to describe the governing equations of reactor
-models as implemented in Cantera.
-
-Wall Interactions
-=================
+.. py:currentmodule:: cantera
 
-At each wall where there are surface reactions, there is a net generation (or
-destruction) of homogeneous phase species. The molar rate of production for
-each species `k` on wall `w` is `\dot{s}_{k,w}`. The total (mass) production
-rate for species `k` on all walls is:
+*****************************
+Reactors and Reactor Networks
+*****************************
 
-.. math::
+A Cantera Reactor represents the simplest form of a chemically reacting system. It corresponds to an extensive thermodynamic control volume `V`, in which all state variables are homogeneously distributed. The system is generally unsteady, i.e. all states are functions of time. In particular, transient state changes due to chemical reactions are possible. However, thermodynamic equilibrium is assumed to be present throughout the reactor at all instants of time.
 
```

Comment:
A homogeneous mixture doesn't necessarily mean that an Equation of State (in whichever form) holds or that there is a defined temperature. Therefore, I thought that it would be good to mention thermodynamic equilibrium. However, I agree that people could be confused between thermodynamic and chemical equilibrium. Do you think that it would be better to write "... thermodynamic (but not necessarily chemical) equilibrium is assumed..."?